api/python/provisioner/inputs.py

```diff
@@ -466,16 +466,84 @@ class Release(ParamGroupInputBase):
     target_build: str = ReleaseParams.target_build
 
 
-class StorageEnclosureParams():
-    _param_group = 'storage_enclosure'
+class ServerDefaultParams():
```

Comment:
@divya-vijay I feel some misunderstanding here: such kind of class helps to avoid copy paste for attributes definitions. 

So the idea is:
- you have some classes `...Params` that just keep attrs definitions and groups them (e.g. network, controller ...)
- you have many validation classes that may include (patially or fully) attrs defined in the classes above

It means:
- we don't need to define attrs multiple time
- in latter classes we may combine attrs from different groups

So here makes sense:
- decide which word to use for node: `server` or `node` (let's say `node`)
- remove `Default` suffix and use `NodeParams` class to define attrs that describes some node
- existent class `NodeNetworkParams` defines set of node params for network, so `NodeParams` may inherit from it and define only non-network params

Reply:
Thanks @andkononykhin for the suggestions. 
Addressed the same for now. And, based on updates/naming convention changes in config design, have planned to finalise then. 

1. For now, since `NodeNetworkParams` had all `srvnode-default` section params, so I have added few more related params to the same class and handled it. 
2. `NodeParams` will consist only of params specific to each node. 
3. `Default` is removed from both Node and StorageEnclosure class definitions.